src/Features/CustomerEffortScoreTracks.php

```diff
@@ -234,4 +269,27 @@ public function run_on_update_options() {
 			)
 		);
 	}
+
+	/**
+	 * Determine on initiating CES survey on searching for product or orders.
+	 */
+	public function run_on_load_edit_php() {
+		$allowed_types = array( 'product', 'shop_order' );
+
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		// We're only interested for certain post types.
+		if ( empty( $_GET['post_type'] ) || ! in_array( $_GET['post_type'], $allowed_types, true ) ) {
+			return;
+		}
+
+		// Determine whether request is search by "s" GET parameter.
+		if ( empty( $_GET['s'] ) ) {
```

Comment:
Could [`get_search_query()`](https://developer.wordpress.org/reference/functions/get_search_query/) be used here?

Reply:
I've tried this, but unfortunately `get_search_query()` returns empty string during our hook call. Bummer 😞